src/plugins/data_source/server/audit/logging_auditor.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuditableEvent, Auditor, Logger, OpenSearchDashboardsRequest } from 'src/core/server';
+
+export class LoggingAuditor implements Auditor {
+  constructor(
+    public readonly request: OpenSearchDashboardsRequest,
```

Comment:
private? not seeing use cases for auditor to expose request to other components

Reply:
updated